WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -587,20 +584,28 @@ class MySiteViewModel @Inject constructor(
         quickStartTask: QuickStartTask,
         position: Int
     ) {
-        if (_activeTaskPosition.value?.first != quickStartTask && isSiteHeaderQuickStartTask(
+        if (_activeTaskPosition.value?.first != quickStartTask && isValidQuickStartFocusPosition(
                         quickStartTask,
                         position
                 )) {
             _activeTaskPosition.postValue(quickStartTask to position)
         }
     }
 
-    private fun isSiteHeaderQuickStartTask(quickStartTask: QuickStartTask, position: Int): Boolean {
-        return if (position == LIST_INDEX_NO_ACTIVE_QUICK_START_ITEM &&
-                (quickStartTask == UPDATE_SITE_TITLE || quickStartTask == UPLOAD_SITE_ICON)) true
+    private fun isValidQuickStartFocusPosition(quickStartTask: QuickStartTask, position: Int): Boolean {
+        return if (position == LIST_INDEX_NO_ACTIVE_QUICK_START_ITEM && isSiteHeaderQuickStartTask(quickStartTask)) true
         else position >= 0
```

Comment:
Nitpick: I got this feedback on one of my PRs, and seeing its benefits in terms of readability sharing it with you:

> When you have an `if/else`, either both are on the same line without brackets or if there are in separate like then preferably add brackets on both.

Reply:
@ashiagr Yup, agree. Will try to keep this in mind and follow it. I have done the changes in [c992906](https://github.com/wordpress-mobile/WordPress-Android/pull/16373/commits/c99290688e62e6c29d90eaa179f1939820de39ee). While formatting, I came across a similar `if-else`which I formatted. It had curly braces in the same line. Let me know if I should revert it.

Reply:
>  While formatting, I came across a similar if-elsewhich I formatted. It had curly braces in the same line. Let me know if I should revert it.

We could have just removed curly braces and kept this `if-else` in one line. 
> When you have an if/else, either both are on the same line without brackets ...

Reply:
👍🏼  Done in [6178ee9](https://github.com/wordpress-mobile/WordPress-Android/pull/16373/commits/6178ee92d7ac970ac59016c5ef05b577083ae28d)